pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -755,18 +754,16 @@ func removeTerminatedContainers(containers []*runtimeapi.Container) []*runtimeap
 	for _, refs := range containerMap {
 		if len(refs) == 1 {
 			result = append(result, refs[0])
```

Comment:
should the above `append` still be present?

Reply:
If the above `append` doesn't need to be present, then this whole if statement can be removed and the block of code below should handle the case where `len(refs) == 1`

Reply:
Oh whoops no, it got accidentally added back when i was cleaning up my branch last night. Good catch, thanks.